### ISSUE TITLE
Unable to parse date format

### ISSUE THREAD

**Issue by martzuk** [20c/pybird#1]
```
>>> status = pybird.get_bird_status()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/pybird.py", line 76, in get_bird_status
    data['last_reboot'] = self._parse_router_status_line(line_iterator.next(), parse_date=True)
  File "/usr/local/lib/python2.7/dist-packages/pybird.py", line 87, in _parse_router_status_line
    return datetime.strptime(data, '%d-%m-%Y %H:%M:%S')
  File "/usr/lib/python2.7/_strptime.py", line 325, in _strptime
    (data_string, format))
ValueError: time data '2017-03-02 13:49:16' does not match format '%d-%m-%Y %H:%M:%S'
```
Seems that the date format may have changed in Bird, I'm using 1.6.3

**Comment by grizz** [comment]
I think this is fixed in HEAD, I'll add a test and do a release if so.

**Comment by grizz** [comment]
Fixed in 1.1.0
